contracts/round/RoundFactory.sol

```diff
@@ -21,111 +21,129 @@ import "../utils/MetaPtr.sol";
+        __Ownable_init_unchained();
+    }
+
+    // --- Core methods ---
+
+    /**
+     * @notice Allows the owner to update the overall protocol fee percentage
+     *
+     * @param newProtocolFeePercentage New protocol fee percentage
+     */
+    function updateProtocolFeePercentage(
+        uint8 newProtocolFeePercentage
+    ) external onlyOwner {
+        protocolFeePercentage = newProtocolFeePercentage;
+
+        emit ProtocolFeePercentageUpdated(protocolFeePercentage);
+    }
+
+    /**
+     * @notice Allows the owner to update the protocol treasury.
+     * This provides us the flexibility to update protocol treasury.
+     *
+     * @param newProtocolTreasury New protocol treasury address
+     */
+    function updateProtocolTreasury(
+        address payable newProtocolTreasury
+    ) external onlyOwner {
+        require(newProtocolTreasury != address(0), "protocolTreasury is 0x");
+
+        protocolTreasury = newProtocolTreasury;
+
+        emit ProtocolTreasuryUpdated(protocolTreasury);
+    }
+
+    /**
+     * @notice Allows the owner to update the RoundImplementation.
+     * This provides us the flexibility to upgrade RoundImplementation
+     * contract while relying on the same RoundFactory to get the list of
+     * rounds.
+     *
+     * @param newRoundContract New RoundImplementation contract address
+     */
+    function updateRoundContract(
+        address payable newRoundContract
+    ) external onlyOwner {
+        require(newRoundContract != address(0), "roundContract is 0x");
+
+        roundContract = newRoundContract;
+
+        emit RoundContractUpdated(newRoundContract);
+    }
+
+    /**
+     * @notice Clones RoundImp a new round and emits event
+     *
+     * @param encodedParameters Encoded parameters for creating a round
+     * @param ownedBy Program which created the contract
+     */
+    function create(
+        bytes calldata encodedParameters,
+        address ownedBy
+    ) external returns (address) {
+        require(roundContract != address(0), "roundContract is 0x");
+        require(protocolTreasury != address(0), "protocolTreasury is 0x");
+
+        address clone = ClonesUpgradeable.clone(roundContract);
+
+        emit RoundCreated(clone, ownedBy, payable(roundContract));
+
+        IRoundImplementation(payable(clone)).initialize(
+            encodedParameters,
+            address(this)
+        );
+
+        return clone;
+    }
+
+        /// @notice Get the address of the RoundImplementation contract
+    function getRoundContract() external view returns (address) {
+        return roundContract;
+    }
+
+    /// @notice Get the address of the protocol treasury
+    function getProtocolTreasury() external view returns (address payable) {
+        return protocolTreasury;
+    }
+
+    /// @notice Get the protocol fee percentage
+    function getProtocolFeePercentage() external view returns (uint8) {
+        return protocolFeePercentage;
+    }
```

Comment:
^ Hmm @KurtMerbeth hmm the relint is making it tricky to review
Could we do the re-lint as a separate PR ? 
We don't have an prettier setup up at the moment from what I can tell